pacehrh/vignettes/pacehrh.rmd

```diff
@@ -260,3 +261,10 @@ sheets and run via an R script.
 
     The results are saved out to a CSV file with the date and name of the geography as specified
     in Excel in the filename. The console will beep at you when the model has finished running. 
+    
+### Install latest version via Github
+
+Alternatively, one can install from repository source code with desired git reference which could be a commit, tag, or branch name
```

Comment:
2 suggestions:
- Move this content to after the plain-vanilla example
- Include examples of installing from a commit and installing from a tag

I would, however, not be unhappy if we just omitted this content and focused entirely on the "tip of main" install.

Reply:
I agree with Charles and would add if we want to keep this, we give some indication of why you might want to install this way. I assume this is primarily for people who want to modify the code and verify behavior after doing so. Minor copyedits:

* Use "GitHub" capitalization
* "with the desired git reference, which" --note added word and comma
* Add a period at the end of the sentence